src/NServiceBus.TransactionalSession.AcceptanceTests/When_using_outbox_send_only.cs

```diff
@@ -128,6 +127,7 @@ public ProcessorEndpoint() => EndpointSetup<DefaultServer>((c, runDescriptor) =>
             }
         );
 
+        //TODO: Discuss if we need this
```

Comment:
@cquirosj this was added when I was trying to find the false positive but I'm on the fence if we still needed it